### PR TITLE
docs: add docs for cleaning up diskspace on the TPP server

### PR DESCRIPTION
**Pull request by bloodearnest**

### CODE REVIEW

File: playbooks/emis-backend.md [hunk 1]
```diff
@@ -1 +1,148 @@
...
+## Configuring the job-runner
+
+All env files are in /srv/jobrunner/environ/\*.env
+
+    01_defaults.env   # job runner default production values. DO NOT EDIT
+    02_secrets.env    # secrets for this backend (e.g. github tokens)
+    03_backend.env    # backend specific configuration. DO NOT EDIT
+    04_local.env      # local overrides - use this to temporarily override config
+
+If you wish to change the config in
```
Comment (StevenMaude) [inline playbooks/emis-backend.md:47]: …?

File: playbooks/emis-backend.md [hunk 2]
```diff
@@ -1 +1,148 @@
...
+
+
+#### Update job-runner
+
+In `/e/job-runner` run:
+
+    git pull
+
+
+Then restart the service
```
Comment (StevenMaude) [inline playbooks/emis-backend.md:59]: nit: "jobrunner service"?

File: playbooks/emis-backend.md [hunk 3]
```diff
@@ -1 +1,148 @@
...
+
+## Switching user
+
+All job-runner operations begin by switching from your user to the jobrunner user with:
+
+    sudo su - jobrunner
+
+This will set up your shell with the correct environment variables.
+
+The jobrunner intalled in `/srv/jobrunner`
```
Comment (StevenMaude) [inline playbooks/emis-backend.md:12]: nit: "is installed" maybe?

NB: typo "intalled".

**Review by StevenMaude — APPROVED** [review]
~From what I can tell, without reviewing fine details on the relevant server, looks good/plausible to me.~

~Just the unfinished sentence that I'd suggest needs fixing :slightly_smiling_face:~

 :arrow_up: was looking at the accidentally included file

All fine :slightly_smiling_face:

File: playbooks/emis-backend.md (hunk 1 above)
Comment (bloodearnest) [reply playbooks/emis-backend.md]: sorry, accidentally included a file

**Pull request merged**